src/module/TaskManagement/src/TaskManagement/Service/MemberOfOrganizationAssertion.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace TaskManagement\Service;
+
+
+use Zend\Permissions\Acl\Acl;
+use Zend\Permissions\Acl\Resource\ResourceInterface;
+use Zend\Permissions\Acl\Role\RoleInterface;
+use Zend\Permissions\Acl\Assertion\AssertionInterface;
+use Ora\User\User;
+
+class MemberOfOrganizationAssertion implements AssertionInterface
+{
+    private $loggedUser;
+    
+    //imposto il default a null su $loggedUser se la richiesta arriva senza che l'utente sia loggato
+    public function __construct(User $loggedUser = null) {
+        $this->loggedUser  = $loggedUser;             
+    }
+    
+	public function assert(Acl $acl, RoleInterface $role = null, ResourceInterface $resource = null, $privilege = null){
+
+		if($this->loggedUser instanceof User){
+			
+		    //controllo se lo stream nel quale creare il task e' associato all'organizzazione dell'utente loggato
+			$currentOrganizationId = $resource->getReadableOrganization();
```

Comment:
Questa variabile non aggiunge valore informativo... se la condizione sotto fosse:
$this->loggedUser->isMemberOf($resource->getOrganization()) direi che esprime benissimo la condizione